sdks/python/apache_beam/examples/complete/estimate_pi_test.py

```diff
@@ -44,7 +44,7 @@ def test_basics(self):
     # Note: Probabilistically speaking this test can fail with a probability
     # that is very small (VERY) given that we run at least 10 million trials.
     assert_that(result, in_between(3.13, 3.15))
-    p.run()
+    p.run().wait_until_finish()
```

Comment:
same (and the rest of the examples below). Perhaps one example should demonstrate waiting...